tests/no_version/test_api_current_user.py

```diff
@@ -141,61 +134,27 @@ async def test_get_current_user_allowed_viewer_paths(
         # Check current-user viewer-paths again
         res = await client.get(f"{PREFIX}allowed-viewer-paths/")
         assert res.status_code == 200
-        assert set(res.json()) == {"/a", "/b", PROJECT_DIR_PLACEHOLDER}
+        assert set(res.json()) == {PROJECT_DIR_PLACEHOLDER}
```

Comment:
Extract allowed-viewer-path into another module, and only keep what is still relevant.